libcodechecker/analyze/analysis_manager.py

```diff
@@ -237,13 +237,30 @@ def check(check_data):
             # sequence, whereas clang-tidy does not. We rewrite the file
             # names to contain no escape sequences for every analyzer.
             result_file = rh.analyzer_result_file.replace(r'\ ', ' ')
+            result_base = os.path.basename(result_file)
+            capture_analysis_output = False
+            if hasattr(args, 'capture_analysis_output'):
```

Comment:
This is woefully redundant here and we don't use this construct anywhere in CodeChecker.

If `default=False` is used, `args.capture_analysis_output` is always defined.

I suggested `default=argparse.SUPPRESS`, for which `'capture_analysis_output' in args` is equivalent to "it is passed and it is `True`".